spec.html

```diff
@@ -122,6 +122,23 @@ <h1>
 		</emu-note>
 	</emu-clause>
 
+	<emu-clause id="sec-create-type-error-copy" type="abstract operation">
+		<h1>
+			CreateTypeErrorCopy (
+				_realmRecord_: a Realm Record,
+				_originalError_: an ECMAScript language value
+			): A TypeError object
+		</h1>
+		<emu-alg>
+			1. Let _newError_ be a newly created *TypeError* object.
+			1. NOTE: _newError_ is created in _realmRecord_.
+			1. Return _newError_.
+		</emu-alg>
+		<p>
```

Comment:
@littledan @mhofman @syg the refactor to introduce the abstract operation is not that important, we have reviewed that already. The import piece is this note.